api/document/serializers/content.py

```diff
@@ -85,10 +86,21 @@ class NestedAnnotationSerializer(serializers.Serializer):
         lambda: BaseAnnotationSerializer,   # will raise an exception when used
     )
 
+    content_type_mapping = {}
+
     def to_representation(self, data: NestableAnnotation):
         serializer = self.serializer_mapping[data.annotation_class]
         return serializer(data, context=self.context).data
 
+    def to_internal_value(self, data: JsonDict) -> JsonDict:
+        content_type = data.get('content_type')
+        if content_type is None:
+            raise ValidationError("missing content_type")
+        if content_type not in self.content_type_mapping:
+            raise ValidationError(f"unknown content_type: {content_type}")
```

Comment:
Should these validations be on the `content_type` field instead? I'd expect cross-field validations to occur here.

Reply:
Yep, see [my other comment](https://github.com/18F/omb-eregs/pull/815#discussion_r160744971) for more details.